config/core/resources/eventtype.yaml

```diff
@@ -155,9 +155,14 @@ spec:
       type: string
       jsonPath: ".status.conditions[?(@.type==\"Ready\")].reason"
   - <<: *version
-    name: v1beta2
+    name: v1beta1
     served: true
-    storage: false
+    storage: true
+    # This indicates the v1beta1 version of the custom resource is deprecated.
+    # API requests to this version receive a warning header in the server response.
+    deprecated: true
+    # This overrides the default warning returned to API clients making v1beta1 API requests.
+    deprecationWarning: "eventing.knative.dev/v1beta1 EventType is deprecated; see https://knative.dev/docs/eventing/event-registry/ for instructions to migrate to eventing.knative.dev/v1beta2 EventType"
```

Comment:
@matzew when I check the link here I couldn't find the instructions to migrate to the next eventtype. Should we add an issue to docs to add this, or am I missing it somewhere?

Reply:
no. it's the regular document, which discusses the ET in detail. that is IMO good enough